src/services/authentication/service.js

```diff
@@ -103,23 +102,26 @@ module.exports = function(app) {
 	}
 
 	function verifyLogin(credentials, systemId) {
-		const systemService = app.service('/systems');
-		return systemService.get(systemId)
+		const System = app.service('/systems').Model;
+		return System.findById(systemId)
+			.then(system => {
+				if(!system) throw new errors.BadRequest(`No system found with id ${systemId}`);
+				return Promise.resolve(system);
+			})
 			.then(system => {
 				return strategies[system.type].login(credentials, system);
 			});
 	}
 
 	function verifyAccount(credentials, systemId) {
-		const accountService = app.service('/accounts');
+		const Account = app.service('/accounts').Model;
```

Comment:
Actually, I think our ifNotLocal hook was a hack. If the feathers people wanted us to use service calls locally, they would have implemented this better, wouldn't they? I think of services like something we offer to other clients